docs/modules/data-structures/partials/nav.adoc

```diff
@@ -40,3 +40,4 @@
 ** xref:data-structures:replicated-map.adoc[]
 ** xref:data-structures:cardinality-estimator-service.adoc[]
 ** xref:data-structures:event-journal.adoc[]
+** xref:data-structures:vector-collections.adoc[Vector Collection]
```

Comment:
Consider moving this up the navigation - probably under Map - for visibility; people are sure to be looking for it following the release!

Reply:
0f509a87194c40b16ed44b3837e9f3d944ca3b42